Jenkinsfile

```diff
@@ -660,9 +660,11 @@ def withBeatsEnv(boolean archive, Closure body) {
   ]) {
     deleteDir()
     unstash 'source'
-    dockerLogin(secret: "${DOCKERELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
-    // FIXME workaround untill we fix the packer cache
-    sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
+    if(os == 'linux'){
```

Comment:
I didn't now `os` was a filter, neat!

Reply:
well, is a tricky function we have in the pipeline of beats, probably we can move it to the library.